aptos-move/framework/supra-framework/sources/poel/i_asset.move

```diff
@@ -0,0 +1,157 @@
+///
+/// This is the iAsset module used as the iAsset standard given to users who deposited assets 
+/// via the intralayer vault. Holders of iAsset get an amount of $supra delegated to staking pools depending on the
+/// amount of assets deposited and the price of the asset. ie If the price of the asset if 
+/// up compared with $Supra the system lends more to the user delegating more $supra 
+/// to the delegation pools and correspondingly if the price drops the system lends less, withdrawing some assets
+/// from the delegation pools. Holders of iAsset get $supra rewards from the staking pools while enabling them to utilize 
+/// the iAsset as an underlying asset of their deposited asset for other defi utilities. Transfer of iAsset to a different address
+/// transfers the acrual of rewards to the receiving address for the period beging after the transfer. 
+///
+
+module supra_framework::i_asset {
+
+    use supra_framework::table;
+    use supra_framework::string::String;
```

Comment:
No such types are available. See `aptos-stdlib` or `move-stdlib` for lib/utility modules available.